Formula/f/fselect.rb

```diff
@@ -51,10 +54,10 @@ def check_binary_linkage(binary, library)
 
     linked_libraries = [
       Formula["libgit2"].opt_lib/shared_library("libgit2"),
+      Formula["libssh2"].opt_lib/shared_library("libssh2"),
       Formula["openssl@3"].opt_lib/shared_library("libcrypto"),
       Formula["openssl@3"].opt_lib/shared_library("libssl"),
```

Comment:
Might just be for the vendored libssh2. If so, can be removed.

Reply:
yeah, just check `Formula["openssl@3"].opt_lib/shared_library("libcrypto"),` for macos builds